### PR TITLE
Eslint compat

### PR DESCRIPTION
**Pull request by expe**
Verifies with OBT v4. Also added a file to ignore bookmarklet.

### CODE REVIEW

File: src/js/getCurrentLayout.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+/**
```
Comment (AlbertoElias) [inline src/js/getCurrentLayout.js]: You created a file that was removed during the merge

**Comment by AlbertoElias** [comment]
You'll need Ruby too so that you can run the tests

**Comment by AlbertoElias** [comment]
Did you checkout the es6 branch? I'm curious what the differences with this one are

**Comment by expe** [comment]
I created the ES6 branch but didn't see this branch had already been created. They were both pretty old.

**Comment by expe** [comment]
It was moved to main.js, I mistakenly re-added it after a merge. See #103.

**Pull request merged**